tools/optrack/find-latency-spikes.py

```diff
@@ -864,7 +864,7 @@ def generateTSSlicesForBuckets():
                               numBuckets, "Generating timeline charts");
 
     for i, fname in returnValues.items():
-        bucketFilenames.append(str(fname.value));
+        bucketFilenames.append(fname.value.decode("ascii"));
```

Comment:
@fedorova, technically, should be `fname.value.decode()` or `fname.value.decode('utf-8')`.  The `fname.value` was generated by this statement at line 685: `retFilename.value = fileName.encode();` - and the default encoding is `utf-8`.  I realize it won't any difference in this case, as they are simple generated file names, but we should be in the habit of matching up encodings/decodings.